language/move-lang/src/parser/syntax.rs

```diff
@@ -523,7 +563,7 @@ fn parse_bind(tokens: &mut Lexer) -> Result<Bind, Error> {
     // The item description specified here should include the special case above for
     // variable names, because if the current tokens cannot be parsed as a struct name
     // it is possible that the user intention was to use a variable name.
-    let ty = parse_module_access(tokens, || "a variable or struct name".to_string())?;
+    let ty = parse_name_access_chain(tokens, || "a variable or struct name")?;
```

Comment:
The grammar comment for `Bind` should refer to `NameAccessChain` instead of `ModuleAccess`